reviewer_aliases.py

```diff
@@ -0,0 +1,18 @@
+# Populate this dict. to add aliases to Github handles when adding reviewers to a PR.
```

Comment:
Thanks for the PR. I'd prefer to figure out how to do this using config files vs. asking users to modify the python file. It'd be useful to be able to configure on a per project/repo basis too.

A couple of ideas. Look for a `.gitworkflow` file in
- `${REPO_ROOT}/.github`
- `${REPO_ROOT}`
- `${HOME}`

We could expect the gitworkflow to be extended with other options/settings in the future.

Reply:
(Oh yikes, sorry for the delay in responding to this.)

That's very fair, and a good idea! Do you have ideas for the syntax of this file, though? Just off the top of my head, I don't have a good idea for
a. how to map handles to aliases in a way that feels like a config language and not python (unless you're suggesting it be python)
b. how to differentiate different kinds of settings in the same file, e.g. if a `.gitworkflow` file contains both a blacklist and an alias mapping. I suppose these could live in separate files, but that sounds like a pain, especially if more settings get added in the future. But I'm also hesitant to describe a whole new syntax, complete with boilerplate to tell what setting you're manipulating... That's why I went with the fill-in-able Python file in the first place: because it seemed easier to give the user a template to fill out (i.e. one that lives in the repo) than to expect them to follow a very specific format on their own. (I don't think this is the _right_ solution, to be clear, but I don't know how to get around this in a `.gitworkflow` file.) Any thoughts?

Reply:
We could use gitconfig for this.

```
[gitworkflow]
	blacklist = opsaccount,troll,bob

[gitworkflow.aliases]
	nick = nicks
	dan = dpup
```

`$ git config gitworkflow.blacklist`
`$ git config gitworkflow.aliases.nick`

Then you can use gitpython for querying the configs.

Reply:
oh, neat! I'll look into it!